src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContext.cs

```diff
@@ -162,6 +162,18 @@ public void RegisterOperationAction(Action<OperationAnalysisContext> action, par
         /// <param name="action">Action to be executed at completion of semantic analysis of an <see cref="IOperation"/>.</param>
         /// <param name="operationKinds">Action will be executed only if an <see cref="IOperation"/>'s Kind matches one of the operation kind values.</param>
         public abstract void RegisterOperationAction(Action<OperationAnalysisContext> action, ImmutableArray<OperationKind> operationKinds);
+
+        /// <summary>
+        /// Register for concurrent execution of analyzer actions registered by this analyzer.
+        /// An analyzer that registers for concurrent execution should be more performant then a non-concurrent analyzer.
```

Comment:
How about "can have better performance than" or "can be more efficient than"? I still haven't accepted "performant" as word--the first definition of it I could find online included "Commonly used by Microsofties but not a real word."

At least, "then" should become "than".